src/sentry/search/events/builder.py

```diff
@@ -194,7 +194,8 @@ def get_snql_query(self) -> Query:
             match=Entity(self.dataset.value),
             select=self.select,
             where=self.where,
-            having=self.having,
+            having=self.having
+            + [Condition(aggregate, Op.IS_NOT_NULL) for aggregate in self.aggregates],
```

Comment:
This is so the aggregates become null in the division right?
Could we just match what we do in the percentage function? eg.
```
if equation.operator == "divide":
    return Function(
        "if",
        [
            Function("greater", [rhs, 0]),
            Function("divide", [lhs, rhs]),
            None
        ],
    )
    ```

Reply:
removed this in favour of returning `None` because this can cause problems with multi y-axis graphs